template/build/webpack.prod.conf.js

```diff
@@ -41,6 +42,22 @@ var webpackConfig = merge(baseWebpackConfig, {
     new ExtractTextPlugin({
       filename: utils.assetsPath('css/[name].[contenthash].css')
     }),
+	new PurifyCSSPlugin({
```

Comment:
Where is this coming from? `PurifyCSSPlugin` is not referenced anywhere, did you forget to import it? Does it not need packages added to `package.json` -- `purifycss-webpack` and `purify-css`?